documentation/source/user_section/tutorials/shape-analysis/normalize-morphometrics-compression.rst

```diff
@@ -1,7 +1,14 @@
 .. _normalize-morphometrics-compressions:
 
+..
+   comment:: In the ``batch_single_subject.sh`` script, this is called
+             "Quantifying spinal cord compression (MSCC) and normalize against
+             database of healthy controls". However, because this page's
+             title gets included in the sidebar and in tabs, it should be
+             kept succinct.
```

Comment:
re: e6dd8539cb5634c58c564dc3329336b7f44d2708; We don't need to keep the bash script titles and tutorial titles 100% consistent, do we?